index.php

```diff
@@ -18,10 +18,33 @@
 
     <script defer src="js/index.js"></script>
 </head>
+<?php
 
-<body>
-
-<h1>Website Template</h1>
+$db = new PDO('mysql:host=db; dbname=collector_app', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+?>
 
+<body>
+    <nav class = 'navbar'>
+        <h2>Footie Shirts: My Collection</h2>
+        <div class = 'nav-links'>
+            <a href="index.php">Home</a>
+            <a href="#">Add</a>
+            <a href="#">Edit</a>
```

Comment:
You should remove these two links for now, you never want to have code in that doesn't fully work

Reply:
done